Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -178,10 +178,15 @@ theorem integrable_add_measure {f : α → β} :
   ⟨fun h => ⟨h.left_of_add_measure, h.right_of_add_measure⟩, fun h => h.1.add_measure h.2⟩
 
 @[simp]
-theorem integrable_zero_measure {_ : MeasurableSpace α} {f : α → β} :
+theorem integrable_zero_measure {f : α → β} :
     Integrable f (0 : Measure α) :=
   ⟨aestronglyMeasurable_zero_measure f, hasFiniteIntegral_zero_measure f⟩
 
+@[fun_prop]
+lemma integrable_dirac [MeasurableSingletonClass α] {a : α} {f : α → β} :
```

Comment:
In other parts of the library, we also have primed versions assuming (in case of `lintegral_dirac`) that `f` is measurable.

Reply:
If it doesn't make sense to add primed versions in this case, then please add comment(s) explaining that.